tests/specs/scoped.js

```diff
@@ -592,7 +592,7 @@ test('scoped Store .off()', function (t) {
   })
 })
 
-test('when type change', function (t) {
+test.only('when type change', function (t) {
```

Comment:
that’s kinda the point, we try to keep it simple for new contributors, and one thing we try to do is to avoid any kind of extra config or dotfiles. So far we went very well with it :) All these rules tend to overtake and the existing maintainers keep adding them to make their live easier, but it makes it harder for new contributors